omero/sysadmins/public.txt

```diff
@@ -101,7 +101,7 @@ To set this up on your OMERO.web installation:
 
     ::
 
-       $ bin/omero config set omero.web.public.url_filter '^/(?!webadmin|webclient/(action|logout|annotate_(file|tags|comment|rating|map)|script_ui|ome_tiff|figure_script)|webgateway/(archived_files|download_as))'
+       $ bin/omero config set omero.web.public.url_filter '^/(webadmin/myphoto/|webclient/(?!(action|annotate_(file|tags|comment|rating|map)|script_ui|ome_tiff|figure_script))|webgateway/(?!(archived_files|download_as)))'
```

Comment:
@will-moore would you like to test this and make sure we are not missing anything?